OpenQA/Isotovideo/CommandHandler.pm

```diff
@@ -119,6 +119,7 @@ sub _send_to_cmd_srv ($self, $data) { myjsonrpc::send_json($self->cmd_srv_fd, $d
 sub _send_to_backend ($self, $data) { myjsonrpc::send_json($self->backend_fd, $data) }
 
 sub send_to_backend_requester ($self, $data) {
+    return undef unless $self->backend_requester;
     myjsonrpc::send_json($self->backend_requester, $data);
```

Comment:
```suggestion
    return undef unless my $requester = $self->backend_requester;
    myjsonrpc::send_json($requester, $data);
```

Reply:
Do I really need a local variable just to access an attribute within the current object?

Reply:
This is faster than accessing it twice (invoking the function that does the attribute lookup twice; there's no inlining) and no additional lines of code are required. So it is in my opinion a win (even if not a big one).

Reply:
It would be still nice to change this.